doc/plugin-criteria.md

```diff
@@ -0,0 +1,51 @@
+Krew plugin criteria
+===
+
+Rationale
+---
+
+Krew is a `kubectl` plugin manager.
+
+A plugin provides a technical solution to at least one _task_ concerning a problem around kubernetes.
+As plugins are invoked via `kubectl <plugin-name>`, they should feel like natural extensions of the `kubectl` command in a way that some kubectl task is extended.
+Stated reversely: without the plugin, users would typically try to solve the plugin task mainly with kubectl.
+
+A plugin should be classified in the following categories:
+
+- Purpose
+- Category
+- Usability
+
+Purpose
+---
+
+This task should be classified in one of the categories:
+
+| #  | Classification                                                                                                                                                                                                                 | Recommendation                                                                                                                                                                                            |
```

Comment:
nit: (hope you won't hate me for saying this) I think if we actually do this part as an HTML table, we can also do 80-char word wrap and maintain this more easily:

```html
<table>
<thead>
<tr>
<td>column</td>
<td>column</td>
</tr>
</thead>
<tbody>
<tr>
<td>column</td>
<td>column</td>
</tr>
</tbody>
</table>
```

Reply:
No, I think this is absolutely necessary. Will do.